mathesar_ui/src/components/cell-fabric/data-types/components/formatted-input/FormattedInputCell.svelte

```diff
@@ -12,6 +12,11 @@
   export let isIndependentOfSheet: $$Props['isIndependentOfSheet'];
   export let formatter: $$Props['formatter'];
   export let formatForDisplay: $$Props['formatForDisplay'];
+
+  $: cssVariables = {
+    ...($$restProps.cssVariables || {}),
+    '--input-element-text-align': 'right',
```

Comment:
```suggestion
    '--input-element-text-align': 'right',
    ...($$restProps.cssVariables ?? {}),
```

I'd like the variable to be placed above the prop, so it can be overridden if required. This change needs to be made on the other components as well.

Reply:
5b0f9ff59